tests/rkt-monitor/main.go

```diff
@@ -283,3 +343,26 @@ func printUsage(statuses []*ProcessStatus) {
 	}
 	fmt.Printf("\n")
 }
+
+func addRecords(statuses []*ProcessStatus, records [][]string) [][]string {
+	for _, s := range statuses {
+		records = append(records, []string{time.Now().String(), s.Name, strconv.Itoa(int(s.Pid)), formatSize(s.RSS), strconv.FormatFloat(s.CPU, 'g', 1, 64)})
+	}
+	return records
+}
+
+func saveRecords(records [][]string, filename string) error {
+	csvFile, err := os.Create(filename)
```

Comment:
This is truncating previous results on multiple runs? For usability, won't you prefer to use an incrementing suffix or let `Tempfile` pick a fresh one for you?